src/Importify/Resolution.hs

```diff
@@ -50,3 +64,15 @@ resolveOneModule m =
         symbolsEnv    = resolve [clearedModule] mempty
         symbols       = concat $ M.elems symbolsEnv
     in symbols
+
+collectUsedQuals :: [ImportDecl SrcSpanInfo] -> [Scoped SrcSpanInfo] -> [ModuleName SrcSpanInfo]
```

Comment:
`collectUnusedSymbols` didn't have *Haddock* documentation either. :) I'll write it for both.